export/imdb/export.py

```diff
@@ -0,0 +1,46 @@
+import os
+import scipy.io
+import asyncio
+from utils.meta_data_parser import mat_to_dict
+from reduct import Client, Bucket
+
+
+REDUCT_STORE_HOST = "https://play.reduct.store"
+REDUCT_STORE_API_TOKEN = os.getenv("REDUCT_STORE_API_TOKEN")
+REDUCT_STORE_BACKET = "datasets"
+REDUCT_STORE_ENTRY = "imdb_"
+
+DATA_SET_PATH = os.getenv('DATA_SET_PATH')
+META_DATA_PATH = os.getenv('META_DATA_PATH')
+
+meta_data = scipy.io.loadmat(META_DATA_PATH)
+meta_data_list = mat_to_dict(meta_data['imdb'])
+
+
+async def main():
+    client = Client(REDUCT_STORE_HOST, api_token=REDUCT_STORE_API_TOKEN)
+    bucket = await client.get_bucket(REDUCT_STORE_BACKET)
+    path = DATA_SET_PATH
+    for foldername in os.listdir(path):
+        dataset_name = foldername
+        counter = 0
+        folder_path = os.path.join(path, dataset_name)
+        for image in os.listdir(folder_path):
+            img_path = os.path.join(folder_path, image)
+            with open(img_path, 'rb') as f:
+                img_data = f.read()
+                for i, img_in_meta_data in enumerate([img['full_path'][3:] for img in meta_data_list]):
+                    if image == img_in_meta_data:
+                        label = meta_data_list[i]
```

Comment:
Didn't manage to make it work. `if image == img_in_meta_data` is always false. I need the images with metadata so if there is no one, then skip image. Moreover, it is very slow. You use a list of dictionaries and search in the loop again and again. It takes 1 second on my laptop. So, the whole export could take 5 days... Try to change your `mat_to_dict` function and return a dictionary where the key is a filename and the value is metadata.